op-node/params/constants.go

```diff
@@ -0,0 +1,7 @@
+package params
+
+const (
+	// Number of L1 blocks between when a channel can be opened and when it must be closed by.
+	ChannelTimeoutBedrock uint64 = 300
+	ChannelTimeoutGranite uint64 = 50
```

Comment:
Is there a reason to type these constants?
```suggestion
	ChannelTimeoutBedrock = 300
	ChannelTimeoutGranite = 50
```

Reply:
It's documentation for alternative clients. It's good to be explicit on the typing of protocol constants and parameters.